apps/google-analytics-4/frontend/src/helpers/signed-requests.ts

```diff
@@ -6,16 +6,16 @@ async function fetchWithSignedRequest(
   appDefinitionId: string,
   cma: PlainClientAPI,
   method: CreateAppSignedRequestProps['method'] = 'GET',
-  headers: Record<string, string> = {}
+  unsignedHeaders: Record<string, string> = {}
```

Comment:
this is still a smell to me. i'm going to work next week on making all of this not an issue on the backend. the signing validation logic should be the only thing responsible for "knowing" what headers matter and don't. Thanks for getting it working with your new additions temporarily either way though 🙇

Reply:
Yeah that's fair. I think we can/shoiuld fold it into a quick (but significant) refactor of this fetching bits. This is what INTEG-137 is about for me. We can strategize about specifics when one of us picks it up.

Reply:
k, sg